docs/jwst/skymatch/description.rst

```diff
@@ -0,0 +1,239 @@
+Description
+===========
+
+Overview
+--------
+The ``skymatch`` step can be used to compute sky values in a collection of input
+images that contain both sky and source signal. The sky values can be computed
+for each image separately or in a way that matches the sky levels amongst the
+collection of images so as to minimize their differences. This operation is
+typically applied before combining multiple images into a mosaic. When running
+the ``skymatch`` step in a matching mode, it compares *total* signal levels
+in *the overlap regions* (instead of doing this comparison on a per-pixel basis)
+of a set of input images and computes the signal offsets for each image that will
+minimize -- in a least squares sense -- the residuals across the entire set.
+This comparison is performed directly on the input images without resampling
+them onto a common grid. The overlap regions are computed directly on the sky
+(celestial sphere) for each pair of input images. By default the sky value
+computed for each image is recorded, but not actually subtracted from the
+images. Also note that the meaning of "sky background" depends on the chosen
+sky computation method.
```

Comment:
True, but I would also add that it is not only sky. There are other contributions, for example thermal self-emission.

Reply:
I agree. In general the name of the method is a bit misleading but if the documentation is clear, it is fine to keep the name as it is.

Reply:
The latest commit mentions the fact that the term "sky" is being used generically in the docs and can include other contributions that just true sky.